### PR TITLE
Minor title changes to reflect the content

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -309,7 +309,7 @@ Legend:
 
 This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no extra delay is induced by the Convert protocol compared to connections that are not proxied. Also, the Convert Protocol does not require any encapsulation (no tunnels, whatsoever).
 
-# ATSSS Version 2
+# ATSSS Version 2: Handling Non-TCP
```
Comment (obonaventure) [inline draft-quic-atsss-reqs.mkd]: Should it be Handling TCP and non-TCP ?

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: Let's keep it simple and focus on non-TCP case to complement what is already supported

Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: Would prefer to mention also 3GPP Rel

e.g. ATSSS Rel. 17: Handling non-TCP

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: I updated the text to introduce 3GPP releases.

Comment (mpiraux) [reply draft-quic-atsss-reqs.mkd]: I feel a unified QUIC solution for TCP and non-TCP flows is envisioned on the mid-term. This impacts the requirements we will be defining later, I would agree to include TCP in that version 2, but maybe this does not correspond to Release 17 yet ?

Comment (markusa) [reply draft-quic-atsss-reqs.mkd]: What's about:

Adding non-TCP support

which leaves open for the time being, if we envisage a unified or complementary solution?

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: > 
> 
> What's about:
> 
> Adding non-TCP support
> 
> which leaves open for the time being, if we envisage a unified or complementary solution?

Deal. Updated the text accordingly.

**Pull request merged**